src/Data/Nat/Binary/Minus.agda

```diff
@@ -0,0 +1,364 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Subtraction on Bin and some of its property proofs.
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Data.Nat.Binary.Minus where
+
+open import Algebra using (Magma)
+import Algebra.FunctionProperties as FuncProp
```

Comment:
`Algebra.FunctionProperties` is deprecated. Please use `Algebra.Definitions` instead.

Reply:
Fixed. And now, `Op₂` is imported from Algebra.